deploy/nginx-plus/deploy.yaml

```diff
@@ -309,16 +312,23 @@ spec:
           name: nginx-run
         - mountPath: /var/cache/nginx
           name: nginx-cache
+        - mountPath: /var/lib/nginx
+          name: nginx-lib
         - mountPath: /etc/nginx/includes
           name: nginx-includes
+        - mountPath: /etc/nginx/license.jwt
```

Comment:
Just for my understanding, we mount the license and secrets to nginx container just for the initial connection? Once we build the first nginx config, we won't use the mounted secrets anymore, right?

Reply:
Ok, we should probably explain that somewhere since it's a little confusing. Maybe a developer doc? We can punt that to 2.0 though.